tests/performace_tests/enkf/test_load_state.py

```diff
@@ -0,0 +1,54 @@
+import pytest
+
+# from libres_utils import ResTest
+from ecl.util.test import ExtendedTestCase
+from ecl.util.util import BoolVector
+from res.test import ErtTestContext
+
+
+@pytest.mark.equinor_test
```

Comment:
Is there any way to create this test without using internal data? Makes it so much easier to debug that way.

Reply:
Test will probably be removed from this PR. See https://github.com/equinor/ert/pull/2913#issuecomment-1042193068